src/jwt/types.ts

```diff
@@ -1,4 +1,4 @@
-import { decode, encode } from "."
+type MaybePromise<T> = T | Promise<T>
```

Comment:
We have a type for this already, called `Awaitable`.

Reply:
fixed